draginowqslb/sensor.go

```diff
@@ -0,0 +1,243 @@
+// Package draginowqslb implements the dragino WQS-LB sensor model
+package draginowqslb
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/viam-modules/gateway/dragino"
+	"github.com/viam-modules/gateway/node"
+	"go.viam.com/rdk/components/sensor"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+)
+
+const (
+	decoderURL = "https://raw.githubusercontent.com/dragino/dragino-end-node-decoder/" +
+		"5a2855dbddba7977e06ba710f33fbee27de124e5/WQS-LB/WQS-LB_ChirpstackV4_Decoder.txt"
```

Comment:
nice we found this one! sucks we can't find the decoder for the other dragino

Reply:
Is this helpful?
https://www.dropbox.com/scl/fo/ztlw35a9xbkomu71u31im/AIeuB8ots6IE61ihETFhC3Y/LoRaWAN%20End%20Node/LHT65N_LHT65S/Decoder?dl=0&preview=LHT65N%20Chirpstack%20%204.0%20decoder.txt&rlkey=ojjcsw927eaow01dgooldq3nu&subfolder_nav_tracking=1

I don't know if their unholy idea of hosting them all on dropbox works as well as a gist.

Reply:
I don't remember which decoder we use currently. can test this one quick next week